### ISSUE TITLE
Is it necessary to excute the codes every time to start davinci

### ISSUE THREAD

**Issue by armornet** [DevMiser/DaVinci#10]
Hello,
       As per your instructions is it necessary to execute the code
python3 DaVinci.py

everytime in the rasberry pi so that the davinci works... 

In your video it seems automatic

**Comment by DevMiser** [comment]
No, you only need to execute the code once and it will run continuously.  What error code do you get when it stops working?  If the error relates to memory, please make certain that you are using a Raspberry Pi 4 and using the 64-bit operating system.
